### PR TITLE
Add new `raw` command to get the raw contents of a message

### PR DESCRIPTION
**Pull request by LewsTherinTelescope**

### CODE REVIEW

**Comment by LewsTherinTelescope** [comment]
metadata I used:
```
{name: 'raw', description: 'Get the raw contents of a message', options: [{ name: 'message', description: 'Link to a message', required: true, type: 3  }] }
```

File: src/raw-message.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { SimpleCommand } from './commands.js';
+
+export default {
+	command: new SimpleCommand(
+		'raw',
+		async (client: Client, interaction: CommandInteraction) => {
+			await interaction.deferReply({ ephemeral: true });
+			const link = interaction.options.getString('message', true);
+			const match = link.match(/(?:discord\.com|discord:\/\/-)\/channels\/(?<guildId>\d{18})\/(?<channelId>\d{18})\/(?<messageId>\d{18})/);
+			if (match) {
```
Comment (PaleoCrafter) [inline src/raw-message.ts:11]: To reduce the amount of nesting in the code, I'd prefer if the "happy" path was the least-indented one. So turn this `if` into a negative test and pull the `else`` contents up here with an early return.

File: src/raw-message.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+								.replaceAll('_', '\\_')
+								.replaceAll('`', '\\`')
+								.replaceAll('|', '\\|')
+								.replaceAll('~', '\\~')
+								.replaceAll('>', '\\>')
+								.replaceAll('<', '\\<')
+						}]
+					});
+				}
+				else {
```
Comment (PaleoCrafter) [inline src/raw-message.ts:33]: I've now enabled the `brace-style` eslint rule on master, so this is no longer valid :sweat_smile:

**Review by PaleoCrafter — CHANGES_REQUESTED** [review]
Code itself looks good, just some minor stylistic things

**Review by PaleoCrafter — APPROVED** [review]

**Pull request merged**